### ISSUE TITLE
.io domains not correctly parsed

### ISSUE THREAD

**Issue by betterphp** [io-developer/php-whois#41]
**PHP-Whois version**: 3.2.2

**PHP version**: 7.2.11

**Description**  
When looking up a .io domain the nameservers are not correctly parsed from the response. For example this is in the whois text
```
Registrant Country: US
Name Server: ALBERT.NS.CLOUDFLARE.COM
Name Server: KRISTIN.NS.CLOUDFLARE.COM
DNSSEC: unsigned

>>> Last update of WHOIS database: 2018-11-11T14:12:00Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information...
```
The parsed result looks like this
```
	"nameservers": {
		"dnssec": "Unsigned",
		"0": "URL of the ICANN WHOIS Data Problem Reporting System: http:\/\/wdprs.internic.net\/",
		"1": ">>> Last update of WHOIS database: 2018-11-11T14:22:56Z <<<"
	},
```

**How to reproduce**  
Lookup a .io domain, I was using codepen.io
```
$whois = Whois::create()->loadDomainInfo("codepen.io");

$nameservers = $whois->getNameservers();

print_r($nameservers);
```

**Comment by io-developer** [comment]
Thanks for report.
Fixed and released (v3.2.3)

**Comment by betterphp** [comment]
Awesome, thanks :)
